.github/workflows/main.yml

```diff
@@ -56,4 +56,4 @@ jobs:
         run: pnpm install
 
       - name: Run lint
-        run: pnpm lint
+        run: pnpm build && pnpm lint
```

Comment:
Note: I realized ESLint emits false positive errors without `pnpm build`. Probably the correct way to fix it to somehow create a command using turborepo for `lint` to depend on `build` but I am not familiar. This should work for now at least for PR pipelines